README.md

```diff
@@ -143,7 +143,7 @@ Please check out `tethys` package object for all available syntax Ops classes
 # derivation
 
 `tethys-derivation` provides semiauto and auto macro derivation JsonReader and JsonWriter instances.  
-In common case you should prefer semiauto derivation because it's more precise, faster in compile and flexible.
+In most cases you should prefer semiauto derivation because it's more precise, faster in compilation and flexible.
```

Comment:
Not sure about this one.
More substitutions: "in general", "generally", "in general case", "in the general case", "in the common case".

Reply:
I am ok with current one